### PR TITLE
Restoring route to `swagger/cromwell.yaml`

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Comment by kshakir** [comment]
Nominating @scottfrazer as first reviewer

**Comment by scottfrazer** [comment]
I have a feeling I removed this code in the first place.... sorry!!!

File: src/main/scala/cromwell/webservice/CromwellApiService.scala [hunk 1]
```diff
@@ -39,6 +39,10 @@ trait CromwellApiService extends HttpService with PerRequestCreator {
   val workflowRoutes = queryRoute ~ workflowOutputsRoute ~ submitRoute ~ workflowStdoutStderrRoute ~ abortRoute ~
     callOutputsRoute ~ callStdoutStderrRoute ~ validateRoute ~ metadataRoute
 
+  def docsRoute = path("swagger" / "cromwell.yaml") {
```
Comment (scottfrazer) [inline src/main/scala/cromwell/webservice/CromwellApiService.scala]: I'm a little confused about this `path` declaration... is this to declare the endpoint as `/swagger`?  what's the "cromwell.yaml" part for?

Comment (scottfrazer) [inline src/main/scala/cromwell/webservice/CromwellApiService.scala]: oh wait, the endpoint is `/swagger/cromwell.yaml`?

Comment (kshakir) [inline src/main/scala/cromwell/webservice/CromwellApiService.scala]: yep

Comment (kshakir) [inline src/main/scala/cromwell/webservice/CromwellApiService.scala]: We used to server all of the static `swagger-ui` files, along with json generated by `spray-swagger`. Now we just need this one static yaml.

Comment (scottfrazer) [inline src/main/scala/cromwell/webservice/CromwellApiService.scala]: Gotcha, sorry again for getting to hasty with removing Swagger code!

:+1:

**Comment by scottfrazer** [comment]
@mcovarr second reviewer

**Comment by mcovarr** [comment]
:+1:

**Pull request merged**